subiquity/controllers/filesystem.py

```diff
@@ -391,18 +409,30 @@ def clear(self, obj):
         for subobj in obj.fs(), obj.constructed_device():
             self.delete(subobj)
 
+    def reformat(self, disk):
```

Comment:
reformat operates on blockdevices, hence the .type check for "disk" UUIC.  Would it be useful to change the parameter here to blockdev or something more generic so the logic for type check becomes more clear?

Reply:
Well this is related to the comment thread in that google doc: what does reformat mean for things that are not disks? Currently this branch just wimps out and only defines reformat for disks so the type check is in that sense redundant.